_includes/events/event-time.html

```diff
@@ -1,40 +1,48 @@
 {%- capture start_day -%}{{ time.start | date: "%Y-%m-%d" }}{%- endcapture -%}
 {%- capture end_day -%}{{ time.end | date: "%Y-%m-%d" }}{%- endcapture -%}
-<strong><i class="far fa-clock" aria-hidden="true"></i>{% if page.repeated %} This is a repeating event{% endif %}</strong> <time datetime="{{ time.start | date_to_xmlschema }}" id="start-time">
-</time>
+<strong><i class="far fa-clock" aria-hidden="true"></i>
+    {% if page.repeated %}This is a repeating event{% endif %}
+</strong>
+<time datetime="{{ time.start | date_to_xmlschema }}" id="start-time"></time>
 {% if time.end %}
-&ndash;
-<time datetime="{{ time.end | date_to_xmlschema }}" id="end-time"></time>
+    &ndash;
+    <time datetime="{{ time.end | date_to_xmlschema }}" id="end-time"></time>
 {% endif %}
+
 <script>
-// Add the browser's time zone to be clear
-var tz = Intl.DateTimeFormat().resolvedOptions().timeZone;
+    // Add the browser's time zone to be clear
+    var tz = Intl.DateTimeFormat().resolvedOptions().timeZone;
 
-// The first date we always show the full date
-var options = { weekday: 'long', year: 'numeric', month: 'long', day: 'numeric', hour: "numeric", minute: "numeric"};
-{% if time.start %}var start_date = new Date("{{ time.start | date_to_xmlschema }}");
-start_date = start_date.toLocaleDateString("en-US", options);
-document.getElementById("start-time").textContent = start_date; 
-{% endif %}{% if time.end %}
+    // The first date we always show the full date
+    var options = { weekday: 'long', year: 'numeric', month: 'long', day: 'numeric', hour: "numeric", minute: "numeric"};
+{% if time.start %}
+    var start_date = new Date("{{ time.start | date_to_xmlschema }}");
+    start_date = start_date.toLocaleDateString("en-US", options);
+    document.getElementById("start-time").textContent = start_date;
+{% endif %}
 
-// If the second date is the same day, no need to write it out again
-{% if start_day == end_day %}options = {hour: "numeric", minute: "numeric"};{% endif %}
-var end_date = new Date("{{ time.end | date_to_xmlschema }}");
-end_date = end_date.toLocaleDateString("en-US", options);
-{% if start_day == end_day %}end_date = end_date.split(" ");
+{% if time.end %}
+    // If the second date is the same day, no need to write it out again
+    {% if start_day == end_day %}
```

Comment:
If possible, it's good to put statements like this on one line, because when you look at the rendering it's going to be littered with spaces / newlines.

Reply:
You mean in the rendered html, or the physical page rendering? This is a button so I'm not sure it matters much either way, but I do always forget the html gets all the extra return lines and spaces.

To answer your above question, this isn't a rendered page really. The button allows you to click on it and then pops up with several options, such as Google Calendar, Outlook, Yahoo, etc. To see if it's working correctly, you need to select one of those and see what it tries to put in that Calendar provider in its title. If you do it on the current site https://us-rse.org/events/2022/2022-03-dei-speaker-series/, you only get "Allies, Collaboration " for the title.

Reply:
Err ignore that first bit haha I thought this was the button html. I did specifically look at the spacing difference in the rendered pages after doing this the last time I touched it and didn't notice anything out of the norm. But if we are concerned about the html having a lot of returns and spaces, then I can revert these two you mentioned.

Reply:
Just the rendered html (view source) and what the dom parses - the actual page doesn't care about white space!

Reply:
I'm not overly concerned - it was just a suggestion and you can safely ignore I think. If this were a page rendering with a LOT of content I'd be more insistent about it, but (I just checked) it's just about 3 empty lines, which is totally ok :)

Reply:
Totally understand. It may just be me, I mean I hate seeing generated HTML with a ton of unneed spaces and returns, but I just have a hard time parsing the .html we are editing when these lines seem to run on for 500 characters haha. If you're okay with the three extra returns, I think I'll keep this as is for now. As I'm sure we are going to need to edit it anyway if we ever address the time start/end, repeated time stuff I mentioned in a previous PR (I guess I never made that an issue yet.)

Reply:
Yep I’m ok with it.  👍